lib/interface/collection/album.dart

```diff
@@ -577,25 +577,20 @@ class AlbumScreenState extends State<AlbumScreen>
     widget.album.tracks.sort(
         (first, second) => first.trackNumber.compareTo(second.trackNumber));
     if (isDesktop) {
-      Timer(
-        Duration(milliseconds: 300),
```

Comment:
Maybe adding the delay will be a good idea since `dart:ui` operations or `PaletteGenerator` freezes the GUI for a substantial amount of time causing the animation to freeze as-well.

Reply:
Adding this will solve the freezes GUI, but the delay itself is not ideal,
mabye we can compute the PaletteGenerator (in isolate) at some early point and not when the page opens,
I will check it.

BTW, I can't see any GUI freezes in my pc, so I guess it depends on the machine.